src/lib/FreeC/Environment/Entry.hs

```diff
@@ -1,6 +1,6 @@
--- | This module contains data types that are used to store information
---   about declared functions, (type) variables and (type) constructors
---   in the environment.
+-- | This module contains data types that are used to store information



+--   about declared functions, (type) variables and (type) constructors



+--   in the environment.




```

Comment:
The line endings have to be fixed. Not sure why the pipeline wasn't triggered when this PR was marked as "Ready for Review".

Reply:
I will fix all line endings. Somehow my fix for brittany didn't work aswell.